src/hypertable.c

```diff
@@ -131,16 +131,16 @@ hypertable_id_to_relid(int32 hypertable_id)
 	return relid;
 }
 
-typedef struct ChunkCacheEntry
+typedef struct ChunkStoreEntry
 {
 	MemoryContext mcxt;
 	Chunk	   *chunk;
-} ChunkCacheEntry;
+} ChunkStoreEntry;
 
 static void
-chunk_cache_entry_free(void *cce)
+chunk_subspace_store_entry_free(void *cse)
```

Comment:
naming consistency: `chunk_subspace_store_entry` does not match `ChunkStoreEntry`